src/test/java/org/folio/rest/api/StorageTestSuite.java

```diff
@@ -326,7 +321,7 @@ private static void startVerticle(DeploymentOptions options)
     deploymentComplete.get(30, TimeUnit.SECONDS);
   }
 
-  private static void prepareTenant(String tenantId) {
+/*  private static void prepareTenant(String tenantId) {
```

Comment:
Why is this code commented out rather than removed?

Reply:
Yes, this needs to be removed, thanks.